jshift-kit-spring-boot/src/main/java/io/jshift/springboot/watcher/SpringBootWatcher.java

```diff
@@ -42,6 +28,19 @@
 import org.apache.maven.artifact.DependencyResolutionRequiredException;
 import org.apache.maven.project.MavenProject;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.List;
+import java.util.Properties;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicBoolean;
+
 import static io.jshift.kit.common.util.SpringBootConfigurationHelper.DEV_TOOLS_REMOTE_SECRET;
 
 public class SpringBootWatcher extends BaseWatcher {
```

Comment:
PR says that we are migrating Thorntail but it seems that also Spring Boot. Now it is ok but in the future let's try to stick on  what description of PR says.

Reply:
It doesn't seem to be migrated, just some imports coming from docker-maven-plugin